daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala

```diff
@@ -1536,10 +1535,13 @@ class SBuiltinTest extends AnyFreeSpec with Matchers with TableDrivenPropertyChe
         e"""ANY_EXCEPTION_MESSAGE (to_any_exception @Mod:ExceptionAppend (Mod:ExceptionAppend { front = "Hello", back = "world"}))"""
       ) shouldBe Right(SText("Helloworld"))
       inside(
-        eval(
-          e"""ANY_EXCEPTION_MESSAGE (to_any_exception @'-unknown-package-':Mod:Exception ('-unknown-package-':Mod:Exception {}))"""
+        Try(
```

Comment:
This is related to an important design choice I made in the implementation of `SpeedyTestLib`.

Roughly speaking implements a loop that make possible to feed Speedy interpretation with external resources (contract, package ...). The important question is what we do in case the resource is not available. By design it is not the responsibility of Speedy (nor the Engine) to handle those failing case but to the caller, so in production those kind of "errors" are not covered by the output type of `Speedy.Machine#run`, namely `Either[SError, SValue]`. 

The question is what to do in case: 
1 - We create a new kind of error and wrap the type Speedy output in an extra layer of Either; 
2 - We use the `SErrorCrash` with an adhoc error message;
3 - We crash with an exception.

(2) was how the loop was implemented in `SBuiltinTest`. 
Why implementing the general version I went for (3), See the `IllegalStateException` in SpeedyTestLib.
I disregard (1) because I judge it complexifies too much the tests.
I chose it over (2) because I prefer the `SErrorCrash` come only for the Speedy code, and not the test helper code.

Reply:
makes sense, thanks for the explanation.